ykcompile/src/lib.rs

```diff
@@ -257,31 +257,28 @@ fn local_to_reg_name(loc: &Location) -> &'static str {
 }
 
 /// Compile a TIR trace, returning executable code.
-pub fn compile_trace<TT>(tt: TirTrace) -> CompiledTrace<TT> {
```

Comment:
It's a shame that we lose the type safety here. I wonder if there's another way we could ensure that the trace for one control point isn't used by another control point? Perhaps it's not worth worrying...

Reply:
Would one solution be to make wrappers that have ignored type parameters?

Reply:
The wrapper in `ykrt` does have the type parameter.

Reply:
Cool. I missed that.